### PR TITLE
New levels 12-13

### PR DESCRIPTION
**Pull request by Felienne**
As discussed in #903

### CODE REVIEW

File: coursedata/level-defaults/fr.yaml [hunk 1]
```diff
@@ -64,7 +64,7 @@
             print animaux at random
 3:
     start_code: "print 'Bonjour le monde'"
-    intro_text: |
+    intro_text: |4
```
Comment (dvberkel) [inline coursedata/level-defaults/fr.yaml:67]: Is this intended?

File: coursedata/yaml_transformer.py [hunk 2]
```diff
@@ -18,9 +19,11 @@ def transform_level_defaults(transformations):
         if level == old_level:
           transformed_dict[new_level] = transformed_dict[old_level]
           del transformed_dict[old_level]
+          file_transformed = True
 
-    with open(output_path + yaml_filesname_without_path, 'w') as f:
-      f.write(utils.dump_yaml_rt(transformed_dict))
+    if file_transformed:
```
Comment (dvberkel) [inline coursedata/yaml_transformer.py:24]: You commented on the reason of this guard on line 49. Do you want to comment it here as well?

File: hedy.py [hunk 3]
```diff
@@ -1093,10 +1102,10 @@ def ask(self, args):
 
         tryblock = textwrap.dedent(f"""
         try:
-          prijs = int({var})
+          {var} = int({var})
```
Comment (dvberkel) [inline hedy.py:1105]: was `prijs` eerder verkeerd?

File: hedy.py [hunk 4]
```diff
@@ -1169,6 +1178,50 @@ def assign(self, args):
...
 
+@hedy_transpiler(level=12)
+class ConvertToPython_12(ConvertToPython_11):
+    def process_comparison(self, args, operator):
+
+        # we are generating an fstring now
+        arg0 = process_variable_for_fstring_padded(args[0], self.lookup)
+        arg1 = process_variable_for_fstring_padded(args[1], self.lookup)
+
+        # zfill(100) leftpads variable to length 100 with zeroes (hence the z fill)
```
Comment (dvberkel) [inline hedy.py:1189]: The comment about `zfill` does not make sense without knowing the implementation of `process_variable_for_fstring_padded`

File: hedy.py (hunk 3 above)
Comment (Felienne) [reply hedy.py:1105]: Yes that was a copy-paste mistake (oops)

**Review by dvberkel — APPROVED** [review]
:shipit:

**Pull request merged**